src/TxHistory/ActionsBanner.tsx

```diff
@@ -45,7 +51,7 @@ export const ActionsBanner = () => {
           {!wallet.isReadOnly && <Spacer width={32} />}
 
           <View style={styles.centralized}>
-            <TouchableOpacity style={styles.actionIcon} onPress={navigateTo.receive} testID="receiveButton">
+            <TouchableOpacity style={styles.actionIcon} onPress={onReceive} testID="receiveButton">
```

Comment:
can we leverage the `disabled` prop of `TouchableOpacity` and remove the check inside the handlers?